src/main/java/org/spongepowered/api/world/extent/Extent.java

```diff
@@ -25,14 +25,15 @@
 
 package org.spongepowered.api.world.extent;
 
-import com.flowpowered.math.vector.Vector3i;
 import org.spongepowered.api.block.BlockLoc;
 import org.spongepowered.api.world.weather.WeatherUniverse;
 
+import com.flowpowered.math.vector.Vector3i;
+
 /**
  * Contains blocks, entities, biomes and weather.
  */
-public interface Extent extends EntityUniverse, WeatherUniverse, BlockVolume, BiomeArea {
+public interface Extent extends EntityUniverse, WeatherUniverse, BlockVolume, BiomeArea, TileTickVolume {
```

Comment:
@kenzierocks It can, but I elected to leave it to make it easier to understand Extent.